website/frontend/templates/index.html

```diff
@@ -117,7 +117,7 @@ <h2>{{ _("Plugin Support") }}</h2>
         infoText = "v2.0.1 for Linux Distributions";
 
       } else if (platform === "MacIntel" || platform === "MacPPC") {
-        downloadURL = "ftp://ftp.musicbrainz.org/pub/musicbrainz/picard/MusicBrainz-Picard-2.0.1.dmg";
+        downloadURL = "https://musicbrainz.osuosl.org/pub/musicbrainz/picard/picard-2.0.1.tar.gz/pub/musicbrainz/picard/MusicBrainz-Picard-2.0.1.dmg";
```

Comment:
Wrong replace?

Reply:
Not sure what happened there, pushed a new commit.